pyfmt/__init__.py

```diff
@@ -108,15 +108,15 @@ def pyfmt(
                     filenames_to_skip.extend([item])
             elif os.path.abspath(item) in all_dirs:
                 # Saving all filenames in directory
-                files_in_dir = list()
+                files_in_dir = []
                 for (dirpath, dirnames, filenames) in os.walk(item):
                     for filename in filenames:
-                        if filename.split(".")[-1] == "py" or filename.split(".")[-1] == "pyi":
+                        if filename.split(".")[-1] in ("py", "pyi"):
                             files_in_dir.append(filename)
                 filenames_to_skip.extend(files_in_dir)
             else:
                 print(
-                    'CRITICAL: One of the files or directories marked as skipped not found ("{}").'.format(
+                    f'CRITICAL: One of the files or directories marked as skipped not found ("{item}").'
                         item
                     )
```

Comment:
this is a syntax error

Reply:
what do you mean?   Not sure, maybe one of those recent changes accepted above?  I just ran it as it was, no syntax error

Reply:
oh nvm this is outdated